### ISSUE TITLE
Do not press the a and d keys if the fps is above 100.

### ISSUE THREAD

**Issue by DeadLord1337** [Ranarrr/rN#3]
Is there any way to fix this?

**Comment by DeadLord1337** [comment]
@Ranarrr https://youtu.be/ZQcjCz8cz4I video

**Comment by Ranarrr** [comment]
Are you using "rayish_fix" cvar? is it set to 1? I think that will fix your problem. This cvar is a fix for button input, where whenever sidemove or forwardmove is more than 0, the button flag will be raised for that value of sidemove/forwardmove. If you are not using it, I will see if I can find a fix for it.

**Comment by DeadLord1337** [comment]
Yes, cvar "rayish_fix" set to 1 (In video, too, most) Else set 0, nothing really helps :/

**Comment by Ranarrr** [comment]
Im just brainstorming, but I think it may be because of the flag being reset every frame. CS 1.6 does this, every time you set a button flag like this (pCmd->buttons |= IN_JUMP) it sets it for that frame, and it will get reset next frame, so if you want it to be held you need to set it every frame, thus I believe that at higher FPS, it will maybe not get set. This is only brainstorming, so it might not be true.
To me it works, but im using a different showkeys plugin.

**Comment by Ranarrr** [comment]
Try and find out what showkeys plugin is being used, and I will try with that one and see if I can produce a fix.

**Comment by DeadLord1337** [comment]
@Ranarrr Not my server, Neon Dragon speedrun. ip: 185.37.216.135:27016

**Comment by Ranarrr** [comment]
Im not entirely sure what is happening for you, because it is working for me on that server. Could you post your settings, where this issue occurs?

**Comment by DeadLord1337** [comment]
echo Executing settings...
//Binds
bind "mouse1" "+attack"
bind "mouse2" "+attack2"
bind "mouse3" "+123_gs"
bind "mouse4" "123_strafe_control_helper 0"
bind "mouse5" "123_strafe_control_helper 1"

//Visual
123_esp 0
123_developer 1
123_fps_max 1001

//Strafe
123_strafe_control_helper 1
123_strafe_control_helper_add_strafe 1
123_strafe_control_helper_adelay 2
123_strafe_control_helper_althreshold 0
123_strafe_control_helper_arthreshold 0
123_strafe_control_helper_dir 1
123_strafe_control_helper_dir_auto 0
123_strafe_control_helper_forwardmove 5
123_strafe_control_helper_sidemove 250
123_strafe_control_helper_rayish_fix 1
123_strafe_control_helper_type 2
123_strafe_control_helper_surffix 1
123_strafe_control_helper_surffix_height 25
123_strafe_control_helper_sidemove_random 0
123_strafe_control_helper_forwardmove_random 0

//HNS-KZ
123_gs_max_speed_random 0
123_gs_max_speed 600
123_gs_scroll 0
123_bhop 1
123_bhop_wog 5000
echo Done!

FPS 500-700

**Comment by DeadLord1337** [comment]
It depends on the fps if fps is 100, everything works well, but if fps is greater than 100(100-500), then there are bugs. Maybe it has something to do with ping?

**Comment by MrJim1231** [comment]
Ranarrr  tell me plz .Why your rN.v0.203 (strafe conrol helper) is DETECTED your rAntiCheat .How can i pass protection?

**Comment by omaremadx3** [comment]
is there any video to explain how to work/change hack settings ik u said don't ask stupid q but lmfao idk wtf to do Iam an super big idot :p
